Dockerfile

```diff
@@ -1,20 +1,23 @@
-FROM quay.io/cdis/python:3.7-alpine as base
+FROM quay.io/cdis/python:3.8-buster as base
 
 FROM base as builder
-RUN apk add --no-cache --virtual .build-deps gcc g++ musl-dev libffi-dev libgit2-dev openssl-dev make postgresql-dev git curl
+RUN pip install --upgrade pip poetry
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    gcc g++ musl-dev libffi-dev libgit2-dev libssl-dev make postgresql git curl
 
 ENV CRYPTOGRAPHY_DONT_BUILD_RUST=1
 
-RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
 COPY . /src/
 WORKDIR /src
-RUN python -m venv /env && . /env/bin/activate && $HOME/.poetry/bin/poetry install --no-interaction
+RUN python -m venv /env && . /env/bin/activate && poetry install --no-interaction --no-dev
 
 FROM base
-RUN apk add --no-cache postgresql-libs
-COPY --from=builder /root/.poetry /root/.poetry
 COPY --from=builder /env /env
 COPY --from=builder /src /src
 ENV PATH="/env/bin/:${PATH}"
 WORKDIR /src
-CMD ["sh", "startup_script.sh"]
+
+CMD ["/env/bin/gunicorn", "thor.main:app", "-b", "0.0.0.0:80", "-k", "uvicorn.workers.UvicornWorker"]
+CMD psql -U postgres -c "create database thor_test_tmp"
+CMD python thor/create_all_tables.py
```

Comment:
I'm getting an error where Python can't open this thor/create_all_tables. Changing this to src/thor/create_all_tables solves this problem, but I'm not sure what other side effects it has.

Reply:
those lines are accidentally pasted here, removed the redundant lines, please try now